servers/hello/src/approov-protected-server/token-check/README.md

```diff
@@ -20,15 +20,18 @@ To lock down your API server to your mobile app. Please read the brief summary i
 
 ## How it works?
 
-The API server is very simple and is defined at [src/approov-protected-server/token-check](src/approov-protected-server/token-check), and only responds to the endpoint `/` with this message:
+The sample API exposes the following endpoints:
 
-```json
-{"message": "Hello, World!"}
-```
+* `/hello` - plain text check that the service is alive.
+* `/token` - validates the Approov token and, when the `ipk` claim is present, verifies the Approov installation message signature. Success returns `Good Token`; failures return a `401` with `Invalid Token`.
+* `/token_binding` - echoes success when the `pay` claim matches the concatenated request headers configured for binding.
+* `/ipk_test` - development helper. Without an `ipk` header it generates and logs a fresh P-256 key pair. With an `ipk` header it validates that the provided public key can be decoded.
+* `/ipk_message_sign_test` - accepts a `private-key` (base64 DER) and a `msg` (base64 canonical message) header and returns an ECDSA P-256/SHA-256 raw signature. The scripts call this to create deterministic signatures.
+* `/sfv_test` - parses and reserialises Structured Field Value headers. The OpenResty quickstart invokes this when running `request_tests_sfv.sh`.
```

Comment:
Mention of OpenResty here - should do a search and replace to reference the correct tech.